src/diffusers/loaders/lora.py

```diff
@@ -1339,7 +1368,7 @@ def process_weights(adapter_names, weights):
             )
         set_weights_and_activate_adapters(text_encoder, adapter_names, text_encoder_weights)
 
-    def disable_lora_for_text_encoder(self, text_encoder: Optional["PreTrainedModel"] = None):
+    def disable_lora_for_text_encoder(self, text_encoder: Optional["PreTrainedModel"] = None):  # noqa: F821
```

Comment:
Why the noqa?

Reply:
Some linters complain about `PreTrainedModel` not being preset